src/components/Menu/MenuContent/MenuContent.tsx

```diff
@@ -15,14 +17,19 @@ const MenuContent: FC<MenuContentProps> = ({
 	variants,
 }) => {
 	const bem = bemCls.bind(root);
-	const rootCls = clsx(className, bem('item'), getVariantClasses(root, variants));
+	const rootCls = clsx(
+		className,
+		bem('item'),
+		getVariantClasses(root, variants),
+		'c-menu-content-item'
+	);
 
 	const renderMenuItem = (menuItemInfo: MenuItemInfo, index: number) => {
 		if (renderItem) {
 			return renderItem(menuItemInfo);
 		}
 		return (
-			<div
+			<button
 				className={rootCls}
 				onClick={() => onClick(menuItemInfo.id)}
 				onKeyPress={(e) => (e.key === 'Space' ? onClick(menuItemInfo.id) : () => null)}
```

Comment:
Deze `onKeyPress` is niet meer nodig. De keyboard accessibility wordt by default opgevangen door het `button` component en deze zal dus de `onClick` triggeren.